src/AlgebraicGeometry/Schemes/AffineAlgebraicSet/Objects/Attributes.jl

```diff
@@ -68,7 +68,8 @@ By Hilbert's Nullstellensatz this is a radical ideal.
 !!! note
     This triggers the computation of a radical, which is expensive.
 """
-vanishing_ideal(X::AbsAffineAlgebraicSet) = ambient_closure_ideal(X)
+vanishing_ideal(X::AbsAffineAlgebraicSet) = saturated_ideal(defining_ideal(X))
+#vanishing_ideal(X::AbsAffineAlgebraicSet) = ambient_closure_ideal(X) # TODO: What about this here? Should it also be removed?
```

Comment:
Summoning @simonbrandhorst . See also the next ones below.

Reply:
I would leave vanishing ideal.